src/marqo/tensor_search/api.py

```diff
@@ -80,6 +84,23 @@ def marqo_user_exception_handler(request, exc: MarqoWebError):
         return JSONResponse(content=body, status_code=exc.status_code)
 
 
+@app.exception_handler(RequestValidationError)
+async def validation_exception_handler(request: Request, exc: RequestValidationError) -> JSONResponse:
```

Comment:
We really need this class to be well tested (resilience for missing dictionary keys and attributes). An error in this class results in an unhelpful 500.